charts/refinery/values.yaml

```
@@ -19,10 +19,10 @@ resources:
   limits:
     cpu: 2000m
     # Changing memory limits requires updating config.InMemCollector.MaxAlloc
-    memory: 2Gi
+    memory: 2097152000 # use bytes to allow auto-configuration of `MaxAlloc`
```

2GB is 2147483648 bytes

```suggestion
    memory: 2147483648 # use bytes to allow auto-configuration of `MaxAlloc`
```

---

Yeah, this is another place where I was confused about the difference between Gigabytes and Gibibytes. Thanks for correcting.